{"name": "Administrador acessa a aplicação cadastro de usuário com perfis", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'perfil_de_acesso_pages'\n", "trace": "  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_perfil_de_acesso_steps.py\", line 8, in step_then_acesso_total\n    assert context.perfil_de_acesso_pages.possui_acesso_total(), \"Usuário não possui acesso total ao sistema.\"\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "steps": [{"name": "Given que o usuário está logado como \"Administrador\"", "status": "passed", "steps": [{"name": "Validando que o usuário está logado como Administrador", "status": "passed", "parameters": [{"name": "context", "value": "<behave.runner.Context object at 0x000001A01A4E81A0>"}], "start": 1744138190150, "stop": 1744138190165}], "start": 1744138190149, "stop": 1744138190165}, {"name": "Then eu devo ter acesso total ao sistema", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'perfil_de_acesso_pages'\n", "trace": "  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_perfil_de_acesso_steps.py\", line 8, in step_then_acesso_total\n    assert context.perfil_de_acesso_pages.possui_acesso_total(), \"Usuário não possui acesso total ao sistema.\"\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\mferreio\\OneDrive - NTT DATA EMEAL\\Documents\\__Github_Projetos__\\AutomacaoDeTeste\\venv\\Lib\\site-packages\\behave\\runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "start": 1744138190165, "stop": 1744138190171}, {"name": "And eu devo conseguir cadastrar usuários com perfis", "status": "skipped", "start": 1744138190173, "stop": 1744138190174}], "start": 1744138190148, "stop": 1744138190174, "uuid": "087928e5-4d4a-496c-a1c5-f92a2fa08534", "historyId": "4e869f391d9f3f3ef7b5162211ee3b1f", "fullName": "Gerenciamento de Perfis de Acesso: Administrador acessa a aplicação cadastro de usuário com perfis", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Gerenciamento de Perfis de Acesso"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}